src/example1/Main.java

```diff
@@ -5,39 +5,30 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hello");
-        /*for (int i = 1; i <= 5; i++) {
-            System.out.println(i);
-            try {
-                Thread.sleep(1000);
-            } catch (InterruptedException e) {
 
-            }
-        }
-
-         */
-        // Afiseaza cate Thread-uri sunt active in acest moment
-        System.out.println("Numarul thread-urilor active: "+Thread.activeCount());
+        // Print how many Threads are active now
+        System.out.println("Number of active Threads: "+Thread.activeCount());
```

Comment:
please format the code style using command + alt + L

Reply:
Resolved